pkg/util/metrics/metric_int64.go

```diff
@@ -36,67 +37,90 @@ type Int64MetricRepresentation struct {
 }
 
 // Int64Metric represents an int64 metric.
-type Int64Metric struct {
-	name    string
-	measure *stats.Int64Measure
+// Type alias added for backward compatibility
+type Int64Metric = OTelInt64Metric
+
+type Int64MetricInterface interface {
+	Record(labelValues map[string]string, value int64) error
 }
 
-// NewInt64Metric create a Int64Metric metric, returns nil when viewName is empty.
```

Comment:
It looks like the "returns nil when viewName is empty" behavior wasn't retained - the program exits in this situation in the new implementation. Would it make sense to keep this behavior? Same for the float64 metric.